scrapi/harvesters/stepic.py

```diff
@@ -0,0 +1,93 @@
+"""
+Stepic.org harvester of MOOC-online courses for SHARE Notification Service
+Example API query: https://stepic.org:443/api/lessons/100
+"""
+
+
+from __future__ import unicode_literals
+
+import json
+import logging
+from dateutil.parser import parse
+import pycountry
+
+from scrapi import requests
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+
+logger = logging.getLogger(__name__)
+
+
+def process_owner(owners_id):
+    logger.info(owners_id)
+    resp = requests.get("https://stepic.org:443/api/users/" + str(owners_id)).json()
+    try:
+        person = resp[u'users'][0]
+    except KeyError:
+        person = {u'first_name': 'None', u'last_name': 'None'}
+    owner = {
+        'name': " ".join([person[u'first_name'], person[u'last_name']]),
+        'givenName': person[u'first_name'],
+        'additionalName': '',
+        'familyName': person[u'last_name'],
+        'email': '',
+        'sameAs': [],
+    }
+    return [owner]
+
+
+class StepicHarvester(JSONHarvester):
+    short_name = 'stepic'
+    long_name = 'Stepic.org Online Education Platform'
+    url = 'http://www.stepic.org/lesson'
```

Comment:
This URL should correspond to a splash or info page for the service itself. In this case, I think ```https://www.stepic.org``` is the better choice. (These URLs are used to link to information about the sources we collect metadata from, so they are purely for human consumption).